src/examples/src/widgets/progress/ProgressWithChangingValues.tsx

```diff
@@ -0,0 +1,27 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import { icache } from '@dojo/framework/core/middleware/icache';
+import Progress from '@dojo/widgets/progress';
+import Button from '@dojo/widgets/Button';
```

Comment:
I think this is failing because this should be ` '@dojo/widgets/button';`

Reply:
Thanks for catching that, @JamesLMilner . I've fixed the import and rebased onto latest master. Care to take another look?